summary-visualiser/test.sh

```diff
@@ -45,3 +56,6 @@ smoke_test_scenario "zero_arc_create_and_read"
 smoke_test_scenario "zero_arc_create_data"
 smoke_test_scenario "zome_call_single_value"
 smoke_test_scenario "zero_arc_create_data_validated"
+
+# Finally, lint the generated HTML to ensure it's valid.
+echo "$html_output" | tidy -eq
```

Comment:
```suggestion
echo "$html_output" | tidy -errors -quiet
```

Prefer long-form args in scripts -- more readable.